content/en/tracing/trace_collection/single-step-apm.md

```diff
@@ -59,30 +62,390 @@ For example, for a Docker Linux container:
+           name: gcr.io/datadoghq/agent:latest
+   ```
+
+   Replace `<DATADOG_SITE>` with your [Datadog site][10]. Your site is {{< region-param key="dd_site" code="true" >}}. (Ensure the correct **DATADOG SITE** is selected on this page).
+
+3. Deploy the Datadog Agent with the above configuration file:
+   ```shell
+   kubectl apply -f /path/to/your/datadog-agent.yaml
+   ```
+
+### Helm chart
+
+You can use the Datadog Helm chart to install the Datadog Agent on all nodes in your cluster with a DaemonSet.
+
+#### Prerequisites
+
+- [Helm][1]
+- If this is a fresh install, add the Helm Datadog repo:
+    ```bash
+    helm repo add datadog https://helm.datadoghq.com
+    helm repo update
+    ```
+
+#### Installation
+
+To enable single step instrumentation with Helm:
+
+1. Create an empty `datadog-values.yaml` file. Any parameters not specified in this file default to those set in [`values.yaml`][14].
+2. Create a Kubernetes Secret to store your Datadog [API key][3] and [app key][15]:
+   
+   ```bash
+   kubectl create secret generic datadog-secret --from-literal api-key=$DD_API_KEY --from-literal app-key=$DD_APP_KEY
+   ```
+3. Set the following parameters in your `datadog-values.yaml` to reference the secret and enable single step instrumentation:
+   ```
+   datadog:
+    apiKeyExistingSecret: datadog-secret
+    appKeyExistingSecret: datadog-secret
+    site: <DATADOG_SITE>
+    apm:
+      instrumentation:
+         enabled: true
+   ```
+   Replace `<DATADOG_SITE>` with your [Datadog site][13]. Your site is {{< region-param key="dd_site" code="true" >}}. (Ensure the correct SITE is selected on the right).
+4. Run the following command:
+   ```bash
+   helm install <RELEASE_NAME> \
+    -f datadog-values.yaml \
+    --set targetSystem=<TARGET_SYSTEM> \
+    datadog/datadog
+   ```
+
+- `<RELEASE_NAME>`: Your release name. For example, `datadog-agent`.
+- `<TARGET_SYSTEM>`: The name of your OS. For example, `linux` or `windows`.
+
+**Note**: If you are using Helm `2.x`, run the following:
+   ```bash
+   helm install --name <RELEASE_NAME> \
+    -f datadog-values.yaml \
+    --set targetSystem=<TARGET_SYSTEM> \
+    datadog/datadog
+   ```
+
+[1]: https://v3.helm.sh/docs/intro/install/
+[2]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
+[3]: https://app.datadoghq.com/organization-settings/api-keys
+[4]: https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics
+[5]: /agent/kubernetes/apm?tab=helm
+[6]: /agent/kubernetes/log?tab=helm
+[8]: https://gcr.io/datadoghq
+[9]: https://gallery.ecr.aws/datadog/
+[10]: https://hub.docker.com/u/datadog/
+[11]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/docs/Migration_1.x_to_2.x.md
+[12]: /integrations/kubernetes_state_core
+[13]: /getting_started/site
+[14]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml
+[15]: https://app.datadoghq.com/organization-settings/application-keys
```

Comment:
Curious if the tab was meant to end here after the links

Reply:
Good question--the tab is supposed to include enabling or disabling namespaces + specifying library versions, because those are only supported for Kubernetes.